deployments/drf_views.py

```diff
@@ -68,6 +70,17 @@
 )
 
 
+class ERUTypeViewset(viewsets.ReadOnlyModelViewSet):
```

Comment:
Hmm this should likely not be a `ReadOnlyModelViewset` as this is not really returning results for a "model".

We just want a pretty simple view where we can get all the various enum options as JSON and return it.

@thenav56 so you think we can just make an entry in `urls.py` and point to a simple function based view for this? Or is there a class based view that seems appropriate?

(Essentially, we want this endpoint to just return a JSON giving all the values of all the ENUMs in the backend).

Reply:
Yeah, a simple view should suffice for this.

Eg.
url: /enum-options/
payload:
```json
{
    "NationalSocietyActionTitle": [
        {
            "key": "national_society_readiness",
            "label": "National Society Readiness"
        },
        ...
    ],
    ...
}
```